app/Controllers/ReservationsController.php

```
@@ -312,6 +316,22 @@ public function createReservation($request, $response, $args)
             }
             $dupStmt->close();
 
+            // Enforce max active loans per user (admin setting; 0 = no limit)
+            $maxLoans = (int) ((new \App\Models\SettingsRepository($this->db))->get('loans', 'max_active_loans_per_user', '0') ?? 0);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '320', column '37'). (undefined)

(MissingImport)